### PR TITLE
Update objectives syntax

### PR DESCRIPTION
**Pull request by enoldev**
This PR:
- Changes the syntax so we're able to include several goals in the same page
- Introduces "flags" (true/false) to control what components we show
- Introduces a new component to show the goals of the section in the "Learning Objectives" page

Demo here: https://royal-frog-0132.on.fleek.co/curriculum/ipfs/objectives/

### CODE REVIEW

**Comment by mrodriguez3313** [comment]
What was our prognosis on this again?

**Comment by mrodriguez3313** [comment]
@walkerlj0 What was our pronosis again? Also @enoldev could you add a description of the work you did, bc I have no idea what is being done here, to be completely honest

**Comment by walkerlj0** [comment]
Following edits done: 
Looks great! It will be great to have everything line up.
 A couple Qs-
Can we do ‘Deep Dive’ instead of all caps like ‘DEEP DIVE’?
Is it possible to make the text a bit more subtle? See changes:
I added in an object for 1.5 just to see

**Comment by walkerlj0** [comment]
Edited and updated
Notes: 
Looks great! It will be great to have everything line up.
 A couple Qs-
Can we do ‘Deep Dive’ instead of all caps like ‘DEEP DIVE’?
Is it possible to make the text a bit more subtle? See changes:
I added in an object for 1.5 just to see

**Comment by walkerlj0** [comment]
A few things- Sorry to be particular, but
Can you either adjust the padding on the icons so they align with the goals (they should lie where red boxes are) or just make them bullet points?
Screen Shot 2022-11-18 at 8 50 53 AM

Remove the duplicate objectives at the end of page, and we should be good to go!
Screen Shot 2022-11-18 at 8 53 50 AM

File: config/_default/goals.json [hunk 1]
```diff
@@ -112,7 +114,34 @@
           "id": "1.45",
           "description": "Discover what advantages Graphsync has over Bitswap"
         }
-      ]
+      ],
+      "levels": ["deep"]
+    },
+    "1.5": {
+      "description": "Understand the IPFS architecture, as well as the ipfs different components and services",
```
Comment (walkerlj0) [inline config/_default/goals.json:121]:
```suggestion
      "description": "Understand the IPFS architecture, as well as the different ipfs components and services",
```

**Comment by enoldev** [comment]
Updated

**Pull request merged**